Locales/koKR.lua

```diff
@@ -1,52 +1,59 @@
 
 if GetLocale() ~= "koKR" then return end
-local _, L = ...
--- L["BCM_AltInvite"] = ""
-L["BCM_AutoLog"] = "자동적으로 대화내용을 기록하며, 레이드 인스던트 던젼에서 전투내용을 기록합니다."
-L["BCM_BNet"] = "귓말이나 대화나 친구접속등의 이벤트에서 괄호와 다양한 ID의 색상을 사용자설정에 맞게 바꿉니다."
-L["BCM_ButtonHide"] = "대화창의 사이드버튼이 필요없는 사람을 위해 완벽하게  숨깁니다."
-L["BCM_ChannelNames"] = "선택적으로 채널이름을 사용자정의화 할 수 있습니다. 예) [파티] >> [P]"
-L["BCM_ChatCopy"] = "This module allows you to copy chat directly from your chat frame by |cFFFFFFFFSHIFT-clicking|r the chat frame tab."
-L["BCM_EditBox"] = "대화입력창을 사용자 설정에 맞게 위로 이동하거나, 배경뒤로 숨기거나 크기를 변경할 수 있습니다."
-L["BCM_Fade"] = "대화창을 마우스로 이동하기전까지 대화내용이 자동적으로 사라집니다."
-L["BCM_Font"] = "글꼴 이름, 크기, 대화창과 입력창의 스타일을 바꿉니다."
-L["BCM_GMOTD"] = "메인대화창에 길드 오늘의 일정을 10초간 다시 표시합니다."
-L["BCM_Highlight"] = "대화창에 당신의 이름이 언급되면 소리를 들려주고, 직업색상을 표시합니다."
-L["BCM_History"] = "대화창에 얼마나 많은 줄의 대화 기록을 기억할지 정합니다."
-L["BCM_InviteLinks"] = "'초대'단어를 스캔하여, 파티 초대를 ALT-클릭 링크로 바꿀 수 있습니다. 예) |cFFFF7256[invite]|r"
--- L["BCM_Justify"] = ""
-L["BCM_PlayerNames"] = "대화창에 플레이어 이름의 레벨,그룹 혹은 괄호의 변경등을 사용자정의로 바꿀 수 있습니다. 예) [85:|cFFFFFFFFCoolPriest|r:5]"
-L["BCM_Resize"] = "대화창을 블리자드가 제공하는 크기보다 더 작게 혹은 크게 바꾸도록 허락합니다."
--- L["BCM_ScrollDown"] = ""
--- L["BCM_Sticky"] = ""
-L["BCM_TellTarget"] = "현재 지정한 대상에서 귓속말을 /tt나 /wt로 보낼 수 있습니다."
-L["BCM_Timestamp"] = "대화창에 시간표시를 바꿉니다. 색상을 바꾸거나 없애거나 시간표시 형식을 바꿀수 있습니다."
-L["BCM_URLCopy"] = "대화창에 링크 가능한 웹사이트주소로 변경합니다. 예)|cFFFFFFFF[www.battle.net]|r"
--- L["BNETICON"] = ""
--- L["BOTTOM"] = ""
-L["CENTER"] = "가운데"
-L["CHANNELNAME"] = "채널 이름"
-L["CHANNELNUMBER"] = "채널 숫자"
-L["CHATLOG"] = "항상 대화내용 기록"
--- L["CLICKTOCOPY"] = ""
-L["COLORMISC"] = "친구나 길드의 직업 색상을 표시합니다."
-L["COMBATLOG"] = "레이드 인스던트 던전에서 전투메세지 기록"
-L["CORE"] = "BasicChatMod를 사용하시는것을 환영합니다. BCM은 대화창을 사용자의 원하는 설정대로 바꿀수 있습니다. BCM은 몇개의 설정이 바뀌면 /reload 하시도록 디자인 되었습니다. /n/n BCM의 기본값은 당신의 대화창을 화면 가장자리에 위치합니다. /n/n BCM은 적은 메모리를 사용하며, 사용하지 않는 모듈은 끌수 있습니다."
-L["CUSTOMCHANNEL"] = "사용자정의 채널"
-L["FAKENAMES"] = "케릭터이름을 실명이름으로 바꿉니다."
-L["GENERAL"] = "일반"
-L["GUILDRECRUIT"] = "길드찾기"
+local _, L
+L["CORE"] = ""BasicChatMods에 오신 걸 환영합니다, 기본 모듈형 접근 방식으로 대화를 사용자 설정합니다. 접근 방식때문에 BCM은 몇몇 변경에 /reload가 필요하도록 설계되었습니다.\n\n기본적으로 BCM은 대화창을 화면 모서리 끝까지 드래그 할 수 있게 해줍니다, 나머지 사용자 설정은 활성화 또는 비활성화 시킬 수 있는 BCM의 모듈로 완료할 수 있습니다.\n\n BCM이 모듈을 비활성하면 메모리를 사용하지 않습니다, 사용하지 않는 건 비활성하세요!"
```

Comment:
You are using 2 quotation marks `""BasicChatMods` you need to remove one so it won't error.